.eslintrc.js

```diff
@@ -2,30 +2,27 @@
 
 module.exports = {
   env: {
-    "node": true
+    node: true,
   },
-  extends: [
-    "eslint:recommended",
-    "plugin:mozilla/recommended",
-  ],
+  extends: ["eslint:recommended", "plugin:mozilla/recommended"],
 
-  plugins: [
-    "mozilla",
-    "json"
-  ],
+  plugins: ["mozilla", "json"],
 
   rules: {
     "babel/new-cap": "off",
     "comma-dangle": ["error", "always-multiline"],
-    "eqeqeq": "error",
-    "indent": ["warn", 2, {SwitchCase: 1}],
+    eqeqeq: "error",
+    indent: ["warn", 2, { SwitchCase: 1 }],
     "mozilla/no-aArgs": "warn",
     "mozilla/balanced-listeners": 0,
     "no-console": "warn",
     "no-debugger": "warn",
     "no-shadow": ["error"],
     "no-unused-vars": "error",
     "prefer-const": "warn",
-    "semi": ["error", "always"],
+    semi: ["error", "always"],
+    "require-jsdoc": "warn",
+    "valid-jsdoc": "warn",
+    "max-len": ["error", 80],
```

Comment:
In PR #107 I changed `"max-len"` to `"warn"` (just earlier today; hadn't looked at that PR in 5 months); I figure that's a rule some people have very strong feelings for/against. Rebasing against the current `develop` with PR #107 merged in should fix this.

Reply:
This rule is anyways removed in #120 since it follows the suggested config from https://github.com/mozilla/shield-studies-addon-template/issues/61 as a base.